docs/config.md

```diff
@@ -5,15 +5,28 @@ These are the config settings available to jj/Jujutsu.
 
 ## Config files and TOML
 
-The config settings are loaded from the following locations. Less common ways to
-specify `jj` config settings are discussed in a later section.
+`jj` loads several types of config settings:
 
-* [The user config file]
-* `.jj/repo/config.toml` (per-repository)
+- The built-in settings. These cannot be edited. They can be viewed in the
+  `cli/src/config/` directory in `jj`'s source repo.
 
-See the [TOML site] and the [syntax guide] for a description of the syntax.
+- The user settings. These can be edited with `jj config edit --user`. User
+settings are located in [the user config file], which can be found with `jj
+config path --user`.
 
-[The user config file]: #user-config-file
+- The repo settings. These can be edited with `jj config edit --repo` and are
+located in `.jj/repo/config.toml`.
+
+- Settings [specified in the command-line](#specifying-config-on-the-command-line).
+
+These are listed in the order they are loaded; the config listed earlier are
+overridden by the config listed later if they disagree. Every type of config
```

Comment:
nit: I feel like s/the config/a config/ on these two lines is more natural (but, as you know, I'm not a native speaker either)

Reply:
Native English speaker here. The reason it is not natural is `config` is not a word, so using `the config are` sounds better than `a config are` to me. Changing it to `the configuration items are`, you can see that the noun and the verb matches. (or `settings are`)

Reply:
I reworded it as "the settings from earlier items...", hope it's less awkward now.